### DISCUSSION TITLE
For tomorrow

### DISCUSSION THREAD

**Discussion by ivavay** [mdwiltfong/secret-santa#21] [Q&A] [answered]
Do I need to make a PR before tomorrow's session so Michael can look through our work?

**Comment by mdwiltfong** [comment, marked as answer]
Hey Ivy!

Thats a good idea in general. Having a PR sitting in the repo allows for
asynchronous work. You can even label it was a WIP.

Michael

On Tue, Dec 12, 2023 at 11:20 PM Ivy Chen ***@***.***> wrote:

> Do I need to make a PR before tomorrow's session so Michael can look
> through our work?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mdwiltfong/secret-santa/discussions/21>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASEVBXLXAWLWSJXILKO5LGLYJEUJPAVCNFSM6AAAAABASQMWYSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHE2TKMJSGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Thank you,
Michael Wiltfong

**Reply by ivavay** [reply]
Okay thanks. Just did!
